src/doc/dev_manual/xmltools.rst

```diff
@@ -0,0 +1,78 @@
+XML Tools
+============================
+
+Overview
+--------------------------------
+VisIt developers use several xml-based code generation tools to implement VisIt's features.
+The source core for these tools is kept in ``src/tools/dev/xml/`` and ``src/tools/dev/xmledit/``.
+ 
+
+CMake Integration
+--------------------
+
+We rely on xml code generation to keep our State object, Attribute, and Plugin APIs up-to-date. To automate the process we provide CMake targets that call our xml code generation tools for each object or plugin registred. Individual code gen targets are all wired into top level targets that allow you to apply
```

Comment:
I thought we had agreed to limit line length in our .rst files.  If I am incorrect, please disregard this comment. Otherwise, please adjust line length appropriately.

Reply:
I don't recall, but happy to do so -- what did we agree on 80 chars?

Reply:
My recollection is that we set an 80 column limit at the first doc hack-a-thon we had in transision from `.doc` to `.rst`. IMHO, 80 column is some kind of a hold-over from the 1970's when we had only 80 column CRT terminals. A larger limit would be better. And, I think there are many places where 80 is not followed now either because someone forgot or because some markdown constructs cannot tolerate line breaks. I wouldn't sweat the line limit here as long as there are indeed lines (in another project I work on a key VIP likes to document a single line to a whole paragraph which simply thwarts standard text diffing tools) and is just a pain to work with.

Reply:
thanks @markcmiller86 -- I did change to 80 col, in this case -- I really i think I needed more lines as well (it was a big long line with multiple sentences)